java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/debugging/launch/NbLaunchRequestHandler.java

```diff
@@ -161,7 +176,7 @@ private static OutputEventArguments convertToOutputEventArguments(String message
     protected void handleTerminatedEvent(DebugAdapterContext context) {
         CompletableFuture.runAsync(() -> {
             try {
-                waitForDebuggeeConsole.get(5, TimeUnit.SECONDS);
+                waitForDebuggeeConsole.get(1, TimeUnit.SECONDS);
```

Comment:
Hi, incidentally I attempted to solve this delay by removing it :) completely in apache/netbeans#2754 . See the discussion in that PR.